### PR TITLE
Feature_SearchBar

### PR DESCRIPTION
**Pull request by moumax**
As a user I want to have an autocompleting search bar.
As a user, I want to have a choice in a list, when I type in my keyboard
As a user when I click on the autocompleted search I get redirected to a page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,5 +17,7 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
```
Comment (lchastanet) [inline package.json:21]: Why ?

**Review by lchastanet — CHANGES_REQUESTED** [review]
Only one little mistake, maybe more changes after merging others pull requests...

File: package.json (hunk 1 above)
Comment (moumax) [reply package.json]: I removed this empty dependency !

**Review by lchastanet — APPROVED** [review]
Nickel

**Pull request merged**